src/LogAnalysis/KnowledgeCSVParser.hs

```diff
@@ -47,20 +46,20 @@ parseErrorCode =
     <|> (string "Unknown"           >> return Unknown)
     <|> (string "Error"             >> return Error)
 
--- |Parse each csv records
-parseKnowledge :: Parser Knowledge -- not really clean code..
+-- | Parse each csv records
+parseKnowledge :: Parser Knowledge  -- not really clean code..
 parseKnowledge = do
     e <- quotedField
-    _ <- char ','
-    _ <- char '"'
+    () <$ char ','
```

Comment:
Revert this, it's really not that readable. There are two options:
- use the previous version - `_ <- char ','`, which I prefer
- use `void` - `void $ char ','`